recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ requirements:
     - pip
   run:
     - python
-    - neotime
+    - neotime <2,>=1.7.1
+    - neobolt <2,>=1.7.4
```

Comment:
This will fail on OS X until https://travis-ci.org/conda-forge/neobolt-feedstock/builds/505763068 finishes.

Reply:
Yeah, I saw that, waiting for neobolt to be published before bumping build_number

Reply:
There is no need to bump the build number. BTW, for new versions it should be reset back to `0`.

Reply:
So how do I rebuild?

Reply:
We can just restart the CIs here.